internal/api/db/handlers/handler.go

```diff
@@ -0,0 +1,11 @@
+package handlers
+
+import "gorm.io/gorm"
+
+type Handler struct {
+	DB *gorm.DB
```

Comment:
I think we can structure this a little better by having constructor functions for each route handler, e.g `NewAddReviewHander(db *DB)` that lets us provide all of the dependencies it needs, and returns the Gin handler.

Reply:
I remove the Handler struct, and define HandlerFunc directly for the routers